peps/pep-0007.rst

```diff
@@ -26,12 +26,21 @@ particular rule:
    clean up someone else's mess (in true XP style).
 
 
-C dialect
-=========
+C standards
+===========
+
+Follow the following standards.
+For features that aren't in the relevant standard (like atomics or some C11
+features in public headers), use CPython-specific wrappers.
+When adding such wrappers, try to make them easy to adjust for unsupported
+compilers.
 
 * Python 3.11 and newer versions use C11 without `optional features
-  <https://en.wikipedia.org/wiki/C11_%28C_standard_revision%29#Optional_features>`_.
-  The public C API should be compatible with C++.
+  <https://en.wikipedia.org/wiki/C11_%28C_standard_revision%29#Optional_features>`__.
+
+* The public C API should be compatible with C99 and C++.
+  (This is more restrictive than what we guarantee users. Remember that this PEP
+  allows exceptions!)
```

Comment:
I don't understand what this means. Maybe a brief example would clarify it?

Reply:
That's a note for users, who sometimes take this out of context. I'll clarify.